extensions/interactions/TextInput/TextInput.html

```diff
@@ -9,9 +27,12 @@
   <div class="row text-input-container">
     <form ng-submit="submitAnswer(answer)" role="form" class="form-horizontal">
       <div class="col-lg-12 col-md-12 col-sm-12">
+        <input ng-hide="rows > 1" type="text" ng-model="answer" class="form-control" placeholder="{{placeholder}}" autofocus/>
```

Comment:
Don't add the autofocus attr. Otherwise when you visit the state with a text field in the editor page, focus will go there automatically, which is wrong.